pkg/ingester/ingester_test.go

```diff
@@ -32,15 +33,22 @@ type testStore struct {
 	chunks map[string][]chunk.Chunk
 }
 
-func newTestStore(t require.TestingT, cfg Config) (*testStore, *Ingester) {
+func newTestStore(t require.TestingT, cfg Config, clientConfig client.Config, limits validation.Limits) (*testStore, *Ingester) {
 	store := &testStore{
 		chunks: map[string][]chunk.Chunk{},
 	}
-	ing, err := New(cfg, store)
+	ing, err := New(cfg, clientConfig, limits, store)
 	require.NoError(t, err)
 	return store, ing
 }
 
+func newTestStoreDefaults(t require.TestingT) (*testStore, *Ingester) {
```

Comment:
Super nit - this reads to me as return the defaults for a test store, not a test store. Maybe `newDefaultTestStore`?

Reply:
I agree. I was trying to keep the first `newTestStore...` so that it would be discoverable for others with type-ahead / auto-complete, but maybe that's misguided. I do prefer the "default" to be a preamble